### PR TITLE
Social networks share mode fix

### PR DESCRIPTION
**Pull request by kancijan**
- removed docs
- fixed share mode bug in the editor where a network toggle would interact with the wrong index in the network's array in cases where not all networks have `url` and `shareUrl`.

### CODE REVIEW

File: blocks/init/src/Blocks/components/social-networks/components/social-networks-editor.js [hunk 1]
```diff
@@ -22,7 +22,10 @@ export const SocialNetworksEditor = (attributes) => {
 		return null;
 	}
 
+	const socialNetworksShareMode = checkAttr('socialNetworksShareMode', attributes, manifest);
 	const socialNetworksNetworks = checkAttr('socialNetworksNetworks', attributes, manifest);
+	const socialNetworksNetworksFiltered = socialNetworksNetworks
+		.filter(({ id }) => socialNetworksShareMode ? manifest.networks[id]?.shareUrl?.length > 0 : manifest.networks[id]?.url?.length > 0);
```
Comment (goranalkovic-infinum) [inline blocks/init/src/Blocks/components/social-networks/components/social-networks-editor.js:28]:
```suggestion
		.filter(({ id }) => socialNetworksShareMode ? manifest.networks?.[id]?.shareUrl?.length > 0 : manifest.networks?.[id]?.url?.length > 0);
```

File: blocks/init/src/Blocks/components/social-networks/components/social-networks-options.js [hunk 2]
```diff
@@ -24,6 +24,8 @@ export const SocialNetworksOptions = (attributes) => {
 
 	const socialNetworksShareMode = checkAttr('socialNetworksShareMode', attributes, manifest);
 	const socialNetworksNetworks = checkAttr('socialNetworksNetworks', attributes, manifest);
+	const socialNetworksNetworksFiltered = socialNetworksNetworks
+		.filter(({ id }) => socialNetworksShareMode ? manifest.networks[id]?.shareUrl?.length > 0 : manifest.networks[id]?.url?.length > 0);
```
Comment (goranalkovic-infinum) [inline blocks/init/src/Blocks/components/social-networks/components/social-networks-options.js:28]:
```suggestion
		.filter(({ id }) => socialNetworksShareMode ? manifest.networks?.[id]?.shareUrl?.length > 0 : manifest.networks?.[id]?.url?.length > 0);
```

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

**Comment by iruzevic** [comment]
why removed docs and storybook?

**Comment by kancijan** [comment]
@iruzevic Goc thought we didn't use it anymore. I'll revert the commit.

**Comment by iruzevic** [comment]
> @iruzevic Goc thought we didn't use it anymore. I'll revert the commit.

We are not using is on projects but for frontend-libs it must be added because without it will not show on our docs page

**Review by iruzevic — APPROVED** [review]

**Pull request merged**